nanobenchmarks/setup_flink.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Latest stable release
+FLINK_VERSION="1.18.0"
+
+# Create new conda env to install PyFlink
+conda create -n raydata-flink pip python=3.10
```

Comment:
oh let's use the same raydata environment to install flink

Reply:
Fixed.